blivet/devicelibs/dasd.py

```diff
@@ -114,6 +114,10 @@ class dasd_info(ctypes.Structure):
 
 def is_ldl_dasd(device):
     """Determine whether or not a DASD is LDL formatted."""
+    statusfile = "/sys/block/%s/device/status" % (device,)
+    if not os.path.isfile(statusfile):
+        return False
```

Comment:
I'd be fine with it if the "test if it's a dasd" were more direct and/or convincing. I just see this causing problems at some point down the road. I'd even be satisfied if the test changed to `if device.startswith("dasd") and ...`